.github/workflows/plugin_test.yaml

```diff
@@ -22,7 +22,7 @@ jobs:
         include:
 
           - plugin: pynxtools-ellips
-            branch: main
+            branch: adjust_lamp_type_for_new_nexus_def2
```

Comment:
Reminder to Revert this before release @lukaspie @RonHildebrandt

Reply:
done